spec/features/admin/sign_in_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Admin SignIn' do
+  let(:admin_user) { create(:admin_user) }
+
+  it 'Admin signs in successfully' do
```

Comment:
Just a suggestion, what if you put this two its inside a `scenario`?

Reply:
<img width="806" alt="Captura de pantalla 2023-12-04 a la(s) 15 38 50" src="https://github.com/rootstrap/rails_api_base/assets/27789496/da5dedf5-4eb5-4ef6-976c-f68e1e6d4d42">
Rubocop doesn't like using Capybara specific methods

Reply:
Actually, yes. I was [taking a look](https://docs.rubocop.org/rubocop-rspec/cops_rspec_capybara.html#rspeccapybarafeaturemethods) at this and there is a way to enable the usage of Capybara-specific words like `feature`, `scenario`, etc; these are disabled to favor the usage of RSpec-specific words instead. The way I found to allow the usage of Capybara-specific words is adding this config:
```
RSpec/Capybara/FeatureMethods:
  EnabledMethods:
    - feature
    - scenario
    - given
    - background
```
Do we want to enable these words? cc @rootstrap/ruby

Reply:
Is there any reasoning about RuboCop defaults?